v2_papers/2009/AD_histogramBased.json

```diff
@@ -0,0 +1,179 @@
+{
+  "version": "v2.1",
+  "reference": {
+    "title": "Histogram-Based Traffic Anomaly Detection",
+    "authors": [
+      "Andreas Kind",
+      "Marc Ph. Stoecklin",
+      "Xenofontas Dimitropoulos"
+    ],
+    "publication_name": "IEEE Transactions on Network Service Management",
+    "publication_type": "peer_reviewed_journal",
+    "year": 2009,
+    "organization_publishers": [
+      "ieee"
+    ],
+    "pages_number_of": 12,
+    "bibtex": {
+      "type": "",
+      "volume": "",
+      "issue": "",
+      "pages": ""
```

Comment:
These bibtex fields must not be empty strings. If there's no content for them, remove them.
"type" field is mandatory though.
For the bibtex fields I recommend copying the values from the bibtex provided by google scholar (google for the paper in scholar, and look at the citation button).